### ISSUE TITLE
Duplicate `token` when copying rows

### ISSUE THREAD

**Issue by liviuconcioiu** [phpmyadmin/phpmyadmin#18694]
### Describe the bug

When a row is copied, the `token` param in the request is duplicated.

### To Reproduce

1. Go to https://demo.phpmyadmin.net/QA_5_2/index.php?route=/sql&db=sakila&table=actor&server=1
2. Copy a row
3. See duplicate token

### Expected behavior

Just one token should be there.

### Screenshots

https://github.com/phpmyadmin/phpmyadmin/assets/25424343/7af6f6f4-738f-47f6-93e6-6f30d8b7c4b9

### Server configuration

- phpMyAdmin version: 5.2.2-dev, 6.0.0-dev

**Comment by liviuconcioiu** [comment]
Another issue with `token` is if the previous phpMyAdmin version is removed and installed a new one (nightly builds), and you browse phpMyAdmin, this message is shown and the second `token` is `undefined`. This is frustrating if a few tabs are open, you have to copy the query and run it again, otherwise you can't browse the tables.

![token1](https://github.com/phpmyadmin/phpmyadmin/assets/25424343/5136b45d-4d1f-4cf0-a54d-6d29bb4162eb)

![token](https://github.com/phpmyadmin/phpmyadmin/assets/25424343/64361544-e096-4b64-a9ff-67b01d74a619)

**Comment by talktogauravsingh** [comment]
@liviuconcioiu Kindly share steps on how to reproduce it.

**Comment by liviuconcioiu** [comment]
> @liviuconcioiu Kindly share steps on how to reproduce it.

Have two different snapshots of phpMyAdmin, one from a few days ago, one from today, with different folder names.

Log in to the first snapshot, browse a table. Rename the folder of the second snapshot with the name of the first snapshot.

Browse the table again and you will see an undefined token and the token mismatch error, and the table can't be browsed.

Another way is to have a cron job that downloads the snapshot every night, and have phpMyAdmin open during the update with a table open.

I've tested your PR, and the undefined token isn't there anymore, but still the table can't be browsed.

**Comment by williamdes** [comment]
Would changing the version in classes/Version do the same to help reproducing the bug?

**Comment by liviuconcioiu** [comment]
> Would changing the version in classes/Version do the same to help reproducing the bug?

Yeah, it can be reproduced this way. I've changed the nightly version from https://github.com/phpmyadmin/phpmyadmin/blob/d59645d8fbabf3ac6b145840cde55d878114c07f/libraries/vendor_config.php#L76.

**Comment by williamdes** [comment]
I confirm this is still a bug, not sure  if it should be fixed for QA (double token). But the double state having two tokens in the JS memory is bad
